setup.py

```diff
@@ -11,5 +11,6 @@
     packages=find_packages(exclude=['tests']),
     install_requires=[
         'python_moztelemetry',  # TODO: pin version
+        'click'
```

Comment:
I think that would be great to add, but it might be nice to get a few more scripts checked-in to get a feel for how multiple click commands integrate together. For the mean while, this script will be launched using `python -m mozetl.topline.historical_backfill <args>`.